thenewboston_node/business_logic/tests/test_memory_blockchain/test_validations/test_misc.py

```diff
@@ -53,8 +51,3 @@ def test_can_validate_blockchain(
     blockchain.add_block(block2)
     blockchain.snapshot_blockchain_state()
     blockchain.validate(is_partial_allowed=False)
-
-
-@pytest.mark.skip('Not implemented yet')
-def test_can_validate_blockchain_in_chunks():
-    raise NotImplementedError()
```

Comment:
Didn't get what should be tested

Reply:
blockchain may grow very large, so it would take a lot of time to validate it in one shot. Therefore we would to be able to validate it by parts (not necessarily equal to file blockchain chunks). This also helps to show progressbar during validation.

Reply:
I will merge this PR. Please, return the unittest back in the other one.